moja-kuhinjica-frontend/src/components/restaurant/restaurant_card/style.js

```diff
@@ -6,7 +6,7 @@ export const Wrapper = styled(ColDiv)`
     background-color: #ffffff;
     box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
     border-radius: 10px;
-    padding: 10px;
+    /* padding: 10px; */
```

Comment:
If commented out, please remove, if this is not doing anything remove comment symbol